ecosystem/ecosystem.html

```diff
@@ -31,8 +31,7 @@ <h1>
 
       <div id="ecosystem-index-cards">
         <div class="list row ecosystem-cards-wrapper cards-right">
-          {% assign ecosystem = site.ecosystem  | sort_natural: "title" %}
-
+          {% assign ecosystem = site.ecosystem | sample: 1000 %}
```

Comment:
> Is there a way to pass in the array size as the sample? That way if they ever go over 100 cards it'll still work.

Did you look into my comment from above? Are we unable to pass in the array size? If that doesn't work and we have to stick with 1000 for now, then this should be good to go.

Reply:
I changed this to use `site.ecosystem.size`